src/foam/nanos/dig/drivers/DigFormatDriver.js

```diff
@@ -270,7 +270,18 @@ foam.CLASS({
         nu = old.fclone();
         nu.copyFrom(obj);
       }
-      return dao.put(nu);
+
+      try {
+        return dao.put(nu);
+      } catch ( ValidationException ve ) {
+        throw new DAOPutException(ve.getMessage());
+      } catch ( CompoundException ce ) {
```

Comment:
Can you comment this block?

Reply:
FObject.validate(x) collects all validation exceptions into a compound exception https://github.com/foam-framework/foam2/blob/aced3a1edf3ec7d8de62f6161b75f37e0cc753a6/src/foam/core/FObject.java#L427-L445 and `getClientRethrowException()` would give the first validation exception from the compound exception.

Reply:
I mean add it as a comment in the code for the benefit of future developers reading the code. Comments here are less useful for them.

Reply:
Added comment in https://github.com/foam-framework/foam2/pull/4772/commits/133dac070ba25f69e7a01f1cd71f8528f9895190.